x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/range_editor.tsx

```diff
@@ -0,0 +1,199 @@
+                  onChange={({ target }) =>
+                    onMaxBarsChange(Number(get(target, 'value', maxHistogramBars)))
+                  }
+                  placeholder={i18n.translate('xpack.lens.indexPattern.ranges.autoIntervals', {
+                    defaultMessage: 'Auto',
+                  })}
+                  showLabels
+                  showInput="inputWithPopover"
+                  prepend={
+                    <>
+                      <EuiText size="s">
+                        {i18n.translate('xpack.lens.indexPattern.ranges.maxIntervals', {
+                          defaultMessage: 'Max intervals',
+                        })}
+                      </EuiText>{' '}
+                      <EuiIconTip
+                        position="right"
+                        content={i18n.translate('xpack.lens.indexPattern.ranges.maxIntervalsHelp', {
+                          defaultMessage:
+                            "Intervals will be selected automatically based on the available data. The maximum number of bars can never be greater than the Advanced Setting's {histogramMaxBars}",
+                          values: { histogramMaxBars: UI_SETTINGS.HISTOGRAM_MAX_BARS },
+                        })}
+                        type="questionInCircle"
+                      />
+                    </>
+                  }
+                />
+              ) : (
+                <EuiFieldNumber
+                  data-test-subj="lns-indexPattern-range-interval-field"
+                  value={interval}
+                  onChange={({ target }) => onIntervalChange(Number(target.value))}
+                  prepend={
+                    <>
+                      <EuiText size="s">
+                        {i18n.translate('xpack.lens.indexPattern.ranges.min', {
+                          defaultMessage: 'Min Interval',
+                        })}
+                      </EuiText>{' '}
+                      <EuiIconTip
+                        position="right"
+                        content={i18n.translate('xpack.lens.indexPattern.ranges.minIntervalsHelp', {
+                          defaultMessage:
+                            "Interval will be automatically scaled in the event that the provided value creates more buckets than specified by Advanced Setting's {histogramMaxBars}",
+                          values: { histogramMaxBars: UI_SETTINGS.HISTOGRAM_MAX_BARS },
+                        })}
+                        type="questionInCircle"
+                      />
+                    </>
+                  }
+                  min={0}
+                />
+              )}
+            </EuiFlexItem>
+          </EuiFlexGroup>
+
+          <EuiButtonEmpty iconType="controlsHorizontal" onClick={() => onToggleEditor()}>
+            {i18n.translate('xpack.lens.indexPattern.ranges.customIntervalsToggle', {
+              defaultMessage: 'Create custom intervals',
+            })}
+          </EuiButtonEmpty>
```

Comment:
This can't be housed in the EuiFormRow.
```suggestion
          <EuiButtonEmpty size="xs" iconType="controlsHorizontal" onClick={() => onToggleEditor()}>
            {i18n.translate('xpack.lens.indexPattern.ranges.customIntervalsToggle', {
              defaultMessage: 'Create custom intervals',
            })}
          </EuiButtonEmpty>
```